tests/integ/modin/sql_counter.py

```diff
@@ -39,7 +36,6 @@
 NO_CHECK = "no_check"
 
 QUERY_COUNT_PARAMETER = "query_count"
```

Comment:
did we remove this param in all tests?

Reply:
I assume this is referring to fallback_count parameter, and yes. See this comment: https://github.com/snowflakedb/snowpark-python/pull/1542#pullrequestreview-2048434019